cachelib/navy/common/Device.cpp

```diff
@@ -310,6 +332,9 @@ class FileDevice : public Device {
   // determine the capacity of an io_uring/libaio queue
   const uint32_t qDepthPerContext_;
 
+  // Device info vector for FDP support
+  const std::vector<std::shared_ptr<FdpNvme>> fdpNvmeVec_{};
+
```

Comment:
Could you move this to right below `fvec_` to match with the initialization order?